### PR TITLE
Snips bashio

### PR DESCRIPTION
**Pull request by Mofeywalker**
Hey @frenck I reworked the code

adresses issue #737

### CODE REVIEW

**Comment by Mofeywalker** [comment]
is there a way to run the shellchecks locally?

**Comment by frenck** [comment]
See shellcheck documentation: https://github.com/koalaman/shellcheck

File: snips/run.sh [hunk 1]
```diff
@@ -1,35 +1,33 @@
-#!/bin/bash
+#!/usr/bin/env bashio
 set -e
 
-CONFIG_PATH=/data/options.json
+ASSISTANT=$(bashio::config 'assistant')
+LANG=$(bashio::config 'language')
+# possibly not needed anymore? can be queried with bashio
+#CUSTOMTTS=$(bashio::config 'custom_tts.active')
```
Comment (pvizeli) [inline snips/run.sh]: please cleanup

Comment (Mofeywalker) [reply snips/run.sh]: I removed the two lines

**Comment by Mofeywalker** [comment]
```bash
[07:58:07] INFO: Starting snips-watch
snips-asr 1.2.1 (0.63.2)
snips-injection 1.2.1 (0.63.2)
snips-audio-server 1.2.1 (0.63.2)
snips-dialogue 1.2.1 (0.63.2)
snips-hotword 1.2.1 (0.63.2)
snips-nlu 1.2.1 (0.63.2) [model_version: 0.19.0]
/snips-entrypoint.sh: line 17: /usr/share/snips/assistant/assistant.json: Permission denied```

So, I tested again and found that I have a permission issue on line 17 in snips-entrypoint.sh, but I have not changed the permissions in comparison to the old code

**Comment by pvizeli** [comment]
Can you rebase the add-on?

**Comment by Mofeywalker** [comment]
I have problems building the plugin locally.

```W: GPG error: https://raspbian.snips.ai/stretch stable InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 531DD1A7B702B14D
W: The repository 'https://raspbian.snips.ai/stretch stable InRelease' is not signed.
Reading package lists...
Building dependency tree...
Reading state information...
E: Version '0.64.0' for 'snips-platform-voice' was not found
E: Version '0.64.0' for 'snips-asr' was not found
E: Unable to locate package snips-injection
E: Version '0.64.0' for 'snips-watch' was not found
E: Version '0.64.0' for 'snips-audio-server' was not found
E: Version '0.64.0' for 'snips-dialogue' was not found
E: Version '0.64.0' for 'snips-hotword' was not found
E: Version '0.64.0' for 'snips-nlu' was not found
E: Version '0.64.0' for 'snips-tts' was not found
```

I use the following command to build an image on my machine:
```docker build --build-arg BUILD_FROM="homeassistant/amd64-base-ubuntu:16.04" --build-arg SNIPS_VERSION="0.64.0" -t local/my-test-addon .```

**Comment by frenck** [comment]
@Mofeywalker If you rebase after #845 is merged, is should be fixed.

**Comment by frenck** [comment]
Looks like the rebase didn't go as planned.

During all this, Sonos has bought Snips and they decided to shut down. We will be deprecating and removing the add-on shortly. Hence, there is not much use on fixing/moving forward with this.

Nevertheless, thank you for your contribution! 👍

**Pull request closed without merging**